tests/sentry/backup/snapshots/fresh_install.json

```diff
@@ -0,0 +1,315 @@
+[
+{
+  "model": "sites.site",
+  "pk": 1,
+  "fields": {
+    "domain": "example.com",
+    "name": "example.com"
+  }
+},
+{
+  "model": "sentry.option",
+  "pk": 1,
+  "fields": {
+    "key": "sentry:last_worker_ping",
+    "last_updated": "2023-06-27T21:40:01.305Z",
+    "last_updated_by": "unknown",
+    "value": 1687902001.2623305
+  }
+},
+{
+  "model": "sentry.option",
+  "pk": 2,
+  "fields": {
+    "key": "sentry:last_worker_version",
+    "last_updated": "2023-06-27T21:40:01.312Z",
+    "last_updated_by": "unknown",
+    "value": "\"23.7.0.dev0\""
+  }
+},
+{
+  "model": "sentry.option",
+  "pk": 3,
+  "fields": {
+    "key": "sentry:install-id",
+    "last_updated": "2023-06-23T00:03:42.785Z",
+    "last_updated_by": "unknown",
+    "value": "\"1bc81f9440424e908a73ca377581cb52682db68a\""
+  }
+},
+{
+  "model": "sentry.option",
+  "pk": 4,
+  "fields": {
+    "key": "sentry:latest_version",
+    "last_updated": "2023-06-27T21:25:02.602Z",
+    "last_updated_by": "unknown",
+    "value": "\"23.6.1\""
+  }
+},
+{
+  "model": "sentry.actor",
+  "pk": 1,
+  "fields": {
+    "type": 0,
+    "user_id": null,
+    "team": 1
+  }
+},
+{
+  "model": "sentry.email",
+  "pk": 1,
+  "fields": {
+    "email": "testing@example.com",
+    "date_added": "2023-06-22T22:59:55.531Z"
+  }
+},
+{
+  "model": "sentry.organization",
+  "pk": 1,
+  "fields": {
+    "name": "Testing",
+    "slug": "testing",
+    "status": 0,
+    "date_added": "2023-06-22T22:54:28.773Z",
+    "default_role": "member",
+    "is_test": false,
+    "flags": "1"
+  }
+},
+{
+  "model": "sentry.user",
+  "pk": 1,
+  "fields": {
+    "password": "pbkdf2_sha256$150000$iEvdIknqYjTr$+QsGn0tfIJ1FZLxQI37mVU1gL2KbL/wqjMtG/dFhsMA=",
```

Comment:
I hope these aren't sensitive!

Reply:
They're not - they're from a local installation of self-hosted done specifically to generate this test data, and definitely not "real" passwords or keys of any kind. I figured it was worthwhile to keep the values that are "shaped" like the real thing.